### PR TITLE
fix load bug

### PR DESCRIPTION
**Pull request by jianwan0214**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #7343 #7342 #7184 #6922 #7617

## What this PR does / why we need it:

### CODE REVIEW

**Review by daviszhen — APPROVED** [review]

File: pkg/vm/process/types.go [hunk 1]
```diff
@@ -114,6 +115,11 @@ type AnalyzeInfo struct {
 	NetworkIO int64
 }
 
+type ElapseInfo struct {
```
Comment (nnsgmsone) [inline pkg/vm/process/types.go:118]: This is what it is used for.

Comment (nnsgmsone) [reply pkg/vm/process/types.go:118]: Shouldn't this be part of the statistical information?

**Review by aunjgr — APPROVED** [review]

File: pkg/vm/process/types.go (hunk 1 above)
Comment (nnsgmsone) [reply pkg/vm/process/types.go:118]: https://github.com/matrixorigin/matrixone/blob/main/proto/plan.proto#L501

**Review by iamlinjunhong — APPROVED** [review]

**Review by ouyuanning — APPROVED** [review]

File: pkg/sql/compile/operator.go [hunk 2]
```diff
@@ -609,6 +611,7 @@ func constructExternal(n *plan.Node, ctx context.Context, fileList []string) *ex
 		Es: &external.ExternalParam{
```
Comment (nnsgmsone) [inline pkg/sql/compile/operator.go:611]: insertNow if you need to execute remotely, please modify the corresponding proto file. thank you

**Review by nnsgmsone — APPROVED** [review]

**Pull request merged**